docs/developer_guide.md

```diff
@@ -171,6 +171,13 @@ go run cmd/main.go \
 ./scripts/update_golden.sh
 ```
 
+### Run import group latency tests
+
+Run the following command in the root directory:
+```bash
```

Comment:
So the normal tests will also run import-group?

Reply:
Split - now we have normal test, and latency tests.

Reply:
Sorry I was confused that latencyTest argument was always false and not passed as flag to the binary?

Reply:
For example, in get_places_in_test.go, line 109 has latencyTest=false, which is the normal golden test. Then on line 110 (inside e2e.TestWithImportGroupLatency), latencyTest=true, which is the latency test. 

So we have both latencyTest = true & false there.

Reply:
Got it... sorry, bad eyes!